### ISSUE TITLE
ExportAsEnums fails with "No parameterless constructor defined for this object" error

### ISSUE THREAD

**Issue by Maximaximum** [reinforced/Reinforced.Typings#66]
I'm sorry for asking such a dumb question, but...
Has anyone ever tested if `ExportAsEnums` works? :)

```VB.net
Public Class ReinforcedTypingsConfiguration
  Public Shared Sub Configure(builder As ConfigurationBuilder)
    builder.ExportAsEnums(
        {GetType(Smth)}
    )
  End Sub
End Class

Public Enum Smth
  One
  Two
  Three
End Enum
```

**Comment by pavel-b-novikov** [comment]
I will double-check tomorrow

**Comment by pavel-b-novikov** [comment]
Fixed in 1.4.92
